lite/examples/object_detection/raspberry_pi/detect_usbwebcam.py

```diff
@@ -0,0 +1,158 @@
+# python3
+#
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Example using TF Lite to detect objects with the Raspberry USB webcam."""
+
+from __future__ import absolute_import
```

Comment:
is this  strictly necessary ? (since you're already targetting python3)

Reply:
I have the same opinion. However, I did not touch the original code as much as I could for the continuity of the referenced **detect_picamera.py**.